mongodbatlas/clusters.go

```diff
@@ -114,61 +114,6 @@ type clustersResponse struct {
 	TotalCount int       `json:"totalCount,omitempty"`
 }
 
-// DefaultDiskSizeGB represents the Tier and the default disk size for each one
-// it can be use like: DefaultDiskSizeGB["AWS"]["M10"]
-var DefaultDiskSizeGB map[string]map[string]float64 = map[string]map[string]float64{
```

Comment:
this was not in use internally and there's no way for you to use it as `mongodbatlas. DefaultDiskSizeGB` so there's no point on keeping something like this
This is the easy approach but I'm happy to revert this or expose it as a `func`

Reply:
Could you remove it again please? it isn't already necessary in the Terraform part.